radius/mods-enabled/eap

```diff
@@ -5,6 +5,7 @@
 		cisco_accounting_username_bug = no
 		max_sessions = 4096
 		tls {
+            tls_min_version = "1.0"
```

Comment:
This isn't quite in line with the lines below.

Reply:
fixed alignment in the rest of the file too

Reply:
Looks like you've removed the `tls_min_version` now

Reply:
yeah put it back in